randovania/games/prime2/patcher/claris_randomizer.py

```diff
@@ -36,7 +36,11 @@ def _get_randomizer_folder() -> Path:
 
 
 def _get_randomizer_path() -> Path:
-    return _get_randomizer_folder().joinpath("Randomizer.exe")
+    nativepath = _get_randomizer_folder().joinpath("randomizer")
```

Comment:
This different path is weird, we'd need to have at least a description of what it is.